src/target/xtensa/xtensa_regs.h

```diff
@@ -40,152 +44,23 @@ enum xtensa_reg_id {
 	XT_REG_IDX_AR13,
 	XT_REG_IDX_AR14,
 	XT_REG_IDX_AR15,
-	XT_REG_IDX_AR16,
```

Comment:
@ianstcdns Can you please clarify, what is the problem with keeping all the definitions here, so that I can use them in the rtos_xxx_stackings?